### DISCUSSION TITLE
Do I need an SSL certificate?

### DISCUSSION THREAD

**Discussion by TheCableGuy99** [AdguardTeam/AdGuardHome#6999] [General]
Hi,

So for all my internal servers I use a reverse proxy, but when it comes to AdGuard Home I've been purchasing my certificate. The reason is because I was unsure if I need an official one or if it will work behind a reverse proxy just fine?

Can anyone tell me if it will work behind a reverse proxy fine for accessing the web interface when my current cert runs out? I cannot see any reason why not but there was a reason I did it this was previously and I cannot remember why if I'm being honest.

Also, are these following settings referring to the web interface or for lookups?

![Screenshot 2024-05-15 095434](https://github.com/AdguardTeam/AdGuardHome/assets/39552752/54a29343-8c13-4d3e-b5f0-b47c61af9321)

Thanks.

**Comment by TheCableGuy99** [comment]
anyone?

**Comment by Valefoth** [comment]
Hi, if it's only for accessing the UI, a wildcard certificate or behing a reverse proxy will do the job.
I have this kind of setup in Kubernetes, however, i'm not using DoH feature, just only basic 53 port udp.

**Comment by TheCableGuy99** [comment]
Ahh finally someone replies, THANK YOU. Yeah I my cert is up for renewal and I wasn't sure if I can just use my reverse proxy now so will go with that. Thank you!
